Assets/MRTK/Core/Definitions/SpatialAwareness/SpatialAwarenessPlanarObject.cs

```diff
@@ -12,6 +13,17 @@ public partial class SpatialAwarenessPlanarObject : BaseSpatialAwarenessObject
         /// </summary>
         public BoxCollider Collider { get; set; }
 
+        private SpatialAwarenessSurfaceTypes planeType = SpatialAwarenessSurfaceTypes.Unknown;
+
+        /// <summary>
+        /// The type of surface (ex: wall) represented by this object. 
+        /// </summary>
+        public SpatialAwarenessSurfaceTypes PlaneType
+        {
+            get => planeType;
+            set => planeType = value;
+        }
```

Comment:
Does this need to be publicly settable? Feels like info the observer would provide when the object is created, as opposed to needing to be settable elsewhere

Reply:
you are probably right :)

Reply:
fixed